kasa/discover.py

```diff
@@ -358,12 +360,19 @@ async def connect_single(
         The device type is discovered by querying the device.
 
         :param host: Hostname of device to query
+        :param port: Optionally set a different port for the device
+        :param credentials: Optionally provide credentials for
+            devices requiring them
+        :param port: Optionally provide an id identifying the protocol
+            to use.  1 is for legacy/original, 2 is for klap
```

Comment:
Fix the docstring, the variable name and the doc is wrong now.

Reply:
Done